internal/service/config_service.go

```diff
@@ -9,6 +9,10 @@ import (
 	"context"
 	"log/slog"
 
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	config2 "github.com/nginx/agent/v3/internal/datasource/config"
```

Comment:
```suggestion
	datasourceConfig "github.com/nginx/agent/v3/internal/datasource/config"
```

Reply:
Renamed to datasource as linter doesn't allow datasourceConfig or datasource_config. We should maybe do soemthing with all the package names being config